### PR TITLE
doc: add link to README to docs for clarity

### PR DESCRIPTION
**Pull request by zr-msft**
Added link to official docs to make it easier to find and navigate to when users come across this project and updated link to ACR docs.

### CODE REVIEW

**Review by FeynmanZhou — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 Notation is a CLI project to add signatures as standard items in the registry ecosystem, and to build a set of simple tooling for signing and verifying these signatures. This should be viewed as similar security to checking git commit signatures, although the signatures are generic and can be used for additional purposes. Notation is an implementation of the [Notary project specifications][notaryv2-specs].
 
+> **IMPORTANT:** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
```
Comment (shizhMSFT) [inline README.md:9]: How about
```suggestion
> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
```

It renders to
> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).

Comment (zr-msft) [reply README.md]: thanks @shizhMSFT

Incorporated your feedback

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by yizha1 — APPROVED** [review]
LGTM, but IANAM
@priteshbandi Looking for your review

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**